inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/flink/deserialization/DeserializationSchema.java

```diff
@@ -181,6 +228,8 @@ public void removeDataFlow(DataFlowInfo dataFlowInfo) throws Exception {
                 multiTenancyDeserializer.removeDataFlow(dataFlowInfo);
                 fieldMappingTransformer.removeDataFlow(dataFlowInfo);
                 recordTransformer.removeDataFlow(dataFlowInfo);
+
+                inLongGroupIdAndStreamIdMap.remove(dataFlowInfo.getId());
```

Comment:
Remove the map between dataFlowId and inlongGroupIdStreamId immediately when the configuration data remove DataFlowInfo, maybe miss some audit data.

Reply:
Once the dataflow is removed, data in memory will be discarded, audit metric is sent only after the correspond data is processed.

Reply:
Then data size in hive is bigger than data size in audit metric, some data have been sent but they do not get the callback.
But the gap can be receivable.